### DISCUSSION TITLE
How to include a private repository in Github Action

### DISCUSSION THREAD

**Discussion by msgui** [apache/incubator-hugegraph#2287] [Q&A]
Question arises from the need to include 'hugegraph-common 1.0.1' from  'stage' repository

1. attempting to modify the Setting.xml file of Maven on Github Action directly (✔ location: $HOME/.m2/settings.xml).
2. resolving the issue of importing dependencies from the 'stage' repository in the POM file.（✔）

**Comment by imbajin** [comment]
@z7658329 do we have the exist issue to record this problem？

refer: 
- https://github.com/apache/incubator-hugegraph/pull/2305

**Reply by msgui** [reply]
I've already resolved it by introducing $HOME, but except for the backend being **memory**, the hugegraph-common:jar:1.0.1 still cannot be located. Finally, the issue is resolved by including the 'stage' repository in the POM.

**Reply by imbajin** [reply]
> I've already resolved it by introducing $HOME, but except for the backend being **memory**, the hugegraph-common:jar:1.0.1 still cannot be located. Finally, the issue is resolved by including the 'stage' repository in the POM.

Note：remember put stage repo in pom is a tricky way，better just for testing
